src/CacheManager.js

```diff
@@ -3,7 +3,12 @@ import * as _ from "lodash";
 import {FileSystem} from "expo";
 import SHA1 from "crypto-js/sha1";
 
-const BASE_DIR = `${FileSystem.cacheDirectory}expo-image-cache/`;
+let _baseDir = `${FileSystem.cacheDirectory}expo-image-cache/`;
```

Comment:
For simplicity, would it be possible to add this method to CacheManager for simplicity?